scripts/e2etests/README.md

```diff
@@ -0,0 +1,28 @@
+# NuGet E2E test scripts
+
+This is a collection scripts for Visual Studio End-to-End tests that run via the Package Manager Console.
+
+The End-to-End test depend on an integration provided by the [API.Test](./../../test/TestExtensions/API.Test/API.Test.csproj) project and the Package Manager Console's access to Visual Studio services.
+
+The [CreateEndToEndTestPackage.ps1](./../cibuild/CreateEndToEndTestPackage.ps1) script generates a zip file that contains all the details needed for running End-to-End test. This zip file can also be generated by the [build.ps1](./../../build.ps1) script.
```

Comment:
I'm hoping to get rid of this script and make it much easier to make the end-to-end test package locally... Maybe don't bother documenting this particular script and I'll add a README to the new MSBuild project?  Now you just `msbuild` it and it builds the dependencies and does everything for you.

Reply:
We can link to that later? 

I'm guessing this PR gets merged prior to the bootstrap one, so you can just tweak it and point to the project readme?